src/style.css

```diff
@@ -0,0 +1,81 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.search-form {
+  position: relative;
+  margin: 20px auto;
+}
+
+.image {
+  position: absolute;
+}
+
+.big-search-logo {
+  height: 19px;
+  width: 19px;
+  left: 25px;
+  top: 25px;
+}
+
+.small-search-logo {
+  height: 11px;
+  width: 11px;
+  left: 13px;
+  top: 15px;
+}
+
+.input {
+  font-family: Avenir, sans-serif;
```

Comment:
add this font family style to `html` and use here `inherit`

Reply:
ok